src/network/org.freedesktop.network1.policy

```diff
@@ -183,4 +183,15 @@
                 <annotate key="org.freedesktop.policykit.owner">unix-user:systemd-network</annotate>
         </action>
 
+        <action id="org.freedesktop.network1.set-persistent-storage">
+                <description gettext-domain="systemd">Set if persistent storage for systemd-networkd is ready</description>
```

Comment:
"Set if" is not a valid grammatical construct.
Maybe "Specify whether persistent storage for systemd-networkd is available."